ambry-replication/src/main/java/com.github.ambry.replication/ReplicationManager.java

```diff
@@ -87,45 +88,53 @@ public RemoteReplicaInfo(ReplicaId replicaId, ReplicaId localReplicaId, Store lo
     initializeTokens(token);
   }
 
-  public ReplicaId getReplicaId() {
+  ReplicaId getReplicaId() {
     return replicaId;
   }
 
-  public ReplicaId getLocalReplicaId() {
+  ReplicaId getLocalReplicaId() {
     return localReplicaId;
   }
 
-  public Store getLocalStore() {
+  Store getLocalStore() {
     return localStore;
   }
 
-  public Port getPort() {
+  Port getPort() {
     return this.port;
   }
 
-  public long getReplicaLagInBytes() {
+  long getRemoteLagFromLocalInBytes() {
```

Comment:
I find remote lag _from_ local (or vice versa) more descriptive and appropriate than remote _to_ local lag in bytes. Is there a reason to you prefer the second one?